core/unit_test/TestReduce.hpp

```diff
@@ -670,6 +670,40 @@ TEST(TEST_CATEGORY, reduction_with_large_iteration_count) {
                           FunctorReductionWithLargeIterationCount(), nu);
   ASSERT_DOUBLE_EQ(nu, double(N));
 }
+
+// For 32-bit builds a View can't store enough elements
+#ifndef KOKKOS_IMPL_32BIT
+void test_reduction_with_large_view() {
+  using ExecutionSpace              = typename TEST_EXECSPACE::execution_space;
+  constexpr long long unsigned size = 1llu << 32;
+  Kokkos::View<int*, TEST_EXECSPACE::memory_space> v(
+      Kokkos::view_alloc(Kokkos::WithoutInitializing, "v"), size);
+
+  Kokkos::parallel_for(
+      Kokkos::RangePolicy<ExecutionSpace,
+                          Kokkos::IndexType<long long unsigned>>(0, size),
+      KOKKOS_LAMBDA(long long unsigned i) { v(i) = 1; });
```

Comment:
Drop that parallel_for and just increment the partial sum below.
```suggestion
```

Reply:
The point was to test `parallel_for` as well.